tests/CMakeLists.txt

```diff
@@ -34,8 +34,9 @@ if(PYTHON_COVERAGE)
 endif()
 
 if(PYTHON_STYLE_TESTS)
-  add_python_style_test(pep8_style_core "${PROJECT_SOURCE_DIR}/romanesco")
-  add_python_style_test(pep8_style_core_tests "${PROJECT_SOURCE_DIR}/tests")
+  add_python_pep8_test(pep8_style_core "${PROJECT_SOURCE_DIR}/romanesco")
```

Comment:
The romanesco code uses `pep8`.  We will have to work on it to get it to pass `flake8`.